arch/models.py

```diff
@@ -46,6 +49,41 @@ def get_absolute_url(self):
     def get_statement(self) -> Optional[str]:
         return get_disk_statement_from_puid(self.puid)
 
+    @property
+    def niceness(self) -> float:
+        total: int = 0
+
+        votes = self.vote_set.all()
```

Comment:
might want to add a type hint for `vote_set` into the `Problem` class so this doesn't fail the type check